libuavcan_drivers/stm32/driver/include/uavcan_stm32/thread.hpp

```diff
@@ -17,6 +17,12 @@
 # include <ctime>
 # include <cstring>
 #elif UAVCAN_STM32_BAREMETAL
+#elif UAVCAN_STM32_FREERTOS
+# include <stm32f4xx.h>
+# include <cmsis_os.h>
+  #ifndef MAX_SEMAPHORE_COUNT
+    #define MAX_SEMAPHORE_COUNT      50
```

Comment:
MAX_SEMAPHORE_COUNT is the maximum count value of the counting semaphore that can be reached. When the semaphore reaches this value it can no longer be 'given'. I assumes that 50 will be enough, if am wrong please tell me how much should i put here.